### ISSUE TITLE
Player stops for demuxed audio if fragments duration is different

### ISSUE THREAD

**Issue by dima-ravnur** [Eyevinn/channel-engine#203]
HLS player stops after 2-3 videos for HLS stream which has demuxed audio with a slightly different #EXINF values: for video it is 6.006 while for audio it is 6.016.
You can reproduce it using these two manifests:
https://amsrmpdevwe-euwe.streaming.media.azure.net/94f0df66-d92d-4d21-815a-756a2a00d186/5882_b4d.ism/manifest(format=m3u8-aapl)
https://amsrmpdevwe-euwe.streaming.media.azure.net/5237a941-7d05-4c56-82bd-c4f51322b43e/5614_0a0.ism/manifest(format=m3u8-aapl)

I've checked videos from your server-demuxed.js demo script and found that audio and video segments durations are equal. So I suspect that there is a root cause. But I am not sure.

**Comment by birme** [comment]
Hi, we are currently working on an issue that might be related, or what do you say @Nfrederiksen ?
https://github.com/Eyevinn/hls-vodtolive/issues/71

**Comment by Nfrederiksen** [comment]
Yes hopefully, our fix, once finished, will resolve this case as well
